### PR TITLE
Made the contributing file a bit more explicit and detailed.

### PR DESCRIPTION
**Pull request by moorepants**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,159 +1,228 @@
...
+Install pip if you don't have it already and then install the dependencies listed
+in the `requirements.txt` file with:
+```
+pip install -r requirements.txt
+```
 
-`notebooks/references.json` 
+## Step 3: Initialize a notebook directory
 
-Simply fill in the notebook id:
```
Comment (etpinard) [inline CONTRIBUTING.md]: Can you undelete this line and put in step 3?

Comment (etpinard) [inline CONTRIBUTING.md]: i.e. put this whole block 

image

in step 3

**Comment by etpinard** [comment]
Thank you very much.

Once the one comment above is taken care of, this will be a great addition to this repo.

**Comment by moorepants** [comment]
I merged the extra info for config.json and references.json into the steps (not all in step 3 though).

**Comment by etpinard** [comment]
Great. Thanks!

**Pull request merged**